build.sbt

```diff
@@ -8,8 +8,8 @@ val compilerOptions = Seq(
 )
 
 inThisBuild(Seq(
-  scalaVersion := "2.12.4",
-  crossScalaVersions := Seq("2.11.12", scalaVersion.value),
+  scalaVersion := "2.13.2",
+  crossScalaVersions := Seq("2.11.8", "2.12.4", scalaVersion.value),
```

Comment:
I would consider dropping the 2.11 version - as it's pretty old now, and usually we only aim to support the current and previous major version.

Reply:
Amigo is still running 2.11. I was thinking we should upgrade that before I pull support from it.